statbot/util.py

```diff
@@ -13,13 +13,28 @@
 import json
 
 __all__ = [
+    'member_update',
     'get_null_id',
     'get_emoji_name',
     'get_emoji_id',
     'embeds_to_json',
     'null_logger',
 ]
 
+def member_update(before, after):
```

Comment:
can we not have more stuff in `util`? this entire module is an antipattern. the function is only used in `sql.py`

(also, can we call this `user_needs_update` or something?)

Reply:
I've renamed it to `member_needs_update`.

I don't see why this module is an antipattern, and it's not used in only `sql.py`. Like for instance the null logger is a concept that several other modules use, so I don't see why it should be duplicated in every module that uses it.